server/src/com/cloud/network/router/NetworkHelperImpl.java

```diff
@@ -755,6 +755,16 @@ protected HypervisorType getClusterToStartDomainRouterForOvm(final long podId) {
         return networks;
     }
 
+    /**
+     * Return a string representing network Cidr for the specifeid network
+     * @param guestNetwork
+     * @return valid network Cidr for the specified network
+     */
+    protected String getValidNetworkCidr(Network guestNetwork){
```

Comment:
I think you should extract this method to another common class (e.g. NetworkModel) since this method can be reused over the different classes.